app/src/main/res/layout/topic_fragment.xml

```diff
@@ -21,20 +21,24 @@
         android:layout_width="match_parent"
         android:layout_height="?attr/actionBarSize"
         android:background="?attr/colorPrimary"
```

Comment:
Also change `colorPrimary` in `colors.xml` file, as this color is currently has issue related to contrast in accessibility scanner. Use: `#00645C` color mentioned in V7 Mocks.

Reply:
You want me to change colorPrimary to #00645C??

Reply:
Yes.

Reply:
used colorPrimaryDark